models/ens/ens_resolver_addresses.sql

```diff
@@ -128,4 +128,5 @@ FROM
         SELECT name, call_tx_hash AS tx_hash, call_block_time AS block_time, address FROM reverse_records
         ) lastone
     ) fin
-WHERE name_rank = 1
\ No newline at end of file
+WHERE name_rank = 1
+AND !(name regexp '.*\\..+\\..*')
```

Comment:
Can I see the example of subdomains not working? I wonder if I can figure out why it's not working